autoload/gofmt.vim

```diff
@@ -66,9 +66,14 @@ function! gofmt#apply() abort
   endif
 
   let input = getline(1, '$') + ['']
-  let diff = split(system(cmd, input), "\n", 1)
+  let output = system(cmd, input)
+  let diff = split(output, "\n", 1)
   if v:shell_error != 0
-    return
+    if g:gofmt_display_errors
```

Comment:
Change this to `if get(g:, 'gofmt_display_errors', 0)`.  Otherwise it will cause an error when the variable doesn't exist.